src/pages/Home/Home.tsx

```diff
@@ -0,0 +1,11 @@
+import Navbar from '@/components/Navbar/Navbar';
```

Comment:
This can still just be " from '@/components' ", i pulled and tested that to make sure

Reply:
Thank you for catching this. I updated this in 7c6c23bd841e0d4068efd7e7a1b199b7ac066088.